### PR TITLE
Wip/tests

### PR DESCRIPTION
**Pull request by leon332157**

### CODE REVIEW

File: src/package.json [hunk 1]
```diff
@@ -5,7 +5,7 @@
     "main": "main.js",
     "scripts": {
         "start": "electron .",
-        "test": "jest __test__ --verbose",
+        "test": "jest __test__ --debug --no-watchman",
```
Comment (EanKeen) [inline src/package.json]: I love the descriptive commit title you have here

Comment (leon332157) [reply src/package.json]: my bad haha

Comment (leon332157) [reply src/package.json]: i was under some stress to make those commit messages

Comment (EanKeen) [reply src/package.json]: Not sure if GitHub mentioned the commit I was on, but was on 21c5f8e3d6f2d20533fcb7c4b4424b96678cdf68. 😛

**Pull request merged**